e2e/protractor/save-remote.js

```diff
@@ -14,7 +16,7 @@ function buildNumber() {
 }
 
 async function uploadScreenshot(alfrescoJsApi, retryCount) {
-    let files = fs.readdirSync(path.join(__dirname, './e2e-output/screenshots'));
+    let files = fs.readdirSync(path.join(__dirname, '../../e2e-output/screenshots'));
```

Comment:
maybe should be using the `projectRoot` variable?
```suggestion
    let files = fs.readdirSync(path.join(projectRoot, 'e2e-output/screenshots'));
```

Reply:
other references to the path can also use single variable